packages/medusa/src/api/routes/admin/batch/complete-batch-job.ts

```diff
@@ -0,0 +1,31 @@
+import { BatchJobService } from "../../../../services"
```

Comment:
Should we have this endpoint. We initially changed away from having the `complete` endpoint and to have the `confirm` endpoint. 
Completing the batch job should not be done by the user but instead be handled by the worker process that is processing the batch job when the job is complete if it's not a dry_run. If it is a dry_run it requires the user to confirm using the `confirm` endpoint to complete the batch job. 

https://github.com/medusajs/medusa/pull/1378#pullrequestreview-946802546